src/ring_theory/hahn_series.lean

```diff
@@ -179,6 +179,16 @@ def single.add_monoid_hom (a : Γ) : R →+ (hahn_series Γ R) :=
 @[simp]
 lemma single.add_monoid_hom_apply {a : Γ} {r : R} : single.add_monoid_hom a r = single a r := rfl
 
+/-- `coeff g` as an additive monoid/group homomorphism -/
+def coeff.add_monoid_hom (g : Γ) : (hahn_series Γ R) →+ R :=
```

Comment:
I think, if you add `@[simps]` to this defn, then it will autogenerate the lemma below. You can check, by writing `@[simps?]`.

Reply:
I've added `@[simps]` to several definitions throughout this file.